doc/docstrings/FacetGrid.ipynb

```diff
@@ -275,8 +275,8 @@
     "g.map_dataframe(sns.scatterplot, x=\"total_bill\", y=\"tip\")\n",
     "g.set_axis_labels(\"Total bill\", \"Tip\")\n",
     "g.fig.subplots_adjust(wspace=0, hspace=0)\n",
-    "for (col_val, row_val), ax in g.axes_dict.items():\n",
-    "    if col_val == \"Lunch\" and row_val == \"Female\":\n",
+    "for (row_val, col_val), ax in g.axes_dict.items():\n",
```

Comment:
axes_dict entries unpack as (row, col) hence the change

Reply:
Ack I screw this up all the time.